commands/cli/parse.go

```diff
@@ -418,7 +418,10 @@ func appendFile(fpath string, argDef *cmds.Argument, recursive, hidden bool) (fi
 	}
 
 	fpath = filepath.ToSlash(filepath.Clean(fpath))
-
+	fpath, err := filepath.EvalSymlinks(fpath)
```

Comment:
I think that this doesnt just resolve top level symlinks.

You could check if the final path element refers to a symlink and resolve it *only* in that case